.gitignore

```diff
@@ -52,3 +52,6 @@ fastlane/Preview.html
 fastlane/screenshots
 /.tmp/
 /.env
+*.keystore
+*.tar
```

Comment:
what is in *.tar?

Reply:
all secured data (key + env+ ...) are compressed before being encrypted using travis command line tool. this is just to be not to commit non encrypted data!

Reply:
ok